documentation/assemblies/oauth/assembly-oauth-authentication.adoc

```diff
@@ -5,15 +5,19 @@
 [id='assembly-oauth-authentication_{context}']
 = Using OAuth 2.0 token-based authentication
 
-Strimzi supports the use of OAuth 2.0 authentication using the _SASL OAUTHBEARER_ mechanism.
+Strimzi supports the use of OAuth 2.0 authentication using the _SASL_OAUTHBEARER_ and _SASL_PLAIN_ mechanisms.
```

Comment:
Why do we call it _SASL_OAUTHBEARER_ and _SASL_PLAIN_ I would everywhere use just _SASL OAUTHBEARER_ and _SASL PLAIN_. Is the form with the underscore used in any SASL, JaaS or Kafka documentation?

Reply:
Underscores are used in the Kafka documentation, for example, https://kafka.apache.org/documentation/#security_sasl_config

@mstruk and @PaulRMellor  - Do you have any objections to me removing the underscores?

Reply:
`SASL_PLAINTEXT` or `SASL_SSL` used there are Kafka's security protocols which (while absolutely confusing) are something different than SASL mechanism you talk about here. I do not think the Kafka doc is using `SASL_PLAIN` or `SASL_OAUTHBEARER`.

Reply:
Checking the Kafka Docs, they mostly use just OAUTHBEARER, sometimes SASL/OAUTHBEARER, and only here and there SASL OAUTHBEARER. In light of that we may simply use OAUTHBEARER, and PLAIN.

Reply:
Great. I'm all for making this simpler.

Reply:
Changed to OAUTHBEARER and PLAIN throughout.

I was careful to NOT change any occurences of `SASL_SSL` and `SASL_PLAINTEXT`.